### ISSUE TITLE
How to make Links to open url in safari?

### ISSUE THREAD

**Issue by MoOx** [iddan/react-universal-markdown#2]
Hi,

Thanks for this awesome lib.
I am trying to open links in safari (usage: react-native, not web (yet)).
I am mostly having hard time to override native "Link" component in order to add a simple onPress with Linking.openURL.
Any tricks on how to do that?
Why it's not possible to pass our own components to override any?

Also any reason Link are not already opening safari by default?

**Comment by MoOx** [comment]
I had to do this

```js
import React, {Component} from 'react';
import PropTypes from 'prop-types';
import { View, StyleSheet, Linking } from 'react-native';
import { NativeMarkdown as Markdown } from 'react-universal-markdown';
import { NativeComponents } from 'react-universal-markdown/src/NativeMarkdown.js';

// ....

// createMarkdownElement method ripped from
// https://github.com/iddan/react-universal-markdown/blob/master/src/NativeMarkdown.js
const createMarkdownElement = render =>
  class extends Component {
    static displayName = render.name;

    static contextTypes = {
      textStyle: PropTypes.array,
    };

    static childContextTypes = {
      textStyle: PropTypes.array,
    };

    getChildContext() {
      return {
        textStyle: [this.context.textStyle, this.props.textStyle],
      };
    }

    render() {
      const {textStyle} = this.context;
      return render({...this.props, textStyle});
    }
  };

NativeComponents.Link = createMarkdownElement(({style, textStyle, destination, children}) => 
  <Text style={[style, textStyle]} onPress={()=> Linking.openURL(destination) }>
    {children}
  </Text>
)
```

Which is a bit weird. I would expect it was possible by passing a prop to Markdown component

**Comment by iddan** [comment]
Hey there! I'm so glad to see more and more use cases for the library.

By now linking was not implemented as it was not required in the context the library has been developed but it is a great feature to add. You're very welcome to propose a pull request.

Also I see extending the components map isn't as easy as it can be. I'll try to provide a better API in the close future.

**Comment by iddan** [comment]
Resolved by #3
